src/components/redux/store.jsx

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] [optional] @team it would be nice if the redux directory was inside the src folder and not the components folder, this makes your code neat and  easier to read